### DISCUSSION TITLE
mt7615e闭源驱动如何添加额外的wifi接口

### DISCUSSION THREAD

**Discussion by lushangkan** [coolsnowwolf/lede#11429] [Q&A / 快速问答]
尝试新建第三个接口是因为在使用中发现5.2Ghz的穿墙信号好，但速度没有5.8Ghz快，而且5.2Ghz的干扰多。

最开始我尝试将另外一个项目的mt7615.3.dat放到`/etc/wireless/mt7615`里，但我尝试在后台启动，结果提示`Please restart manually to create this interface.`。后面使用mtkwifi up手动启动发现原来是没有rax0的wifi接口。

[最开始的闭源驱动](https://github.com/Nossiac/mtk-openwrt-feeds)是有ra0 rai0 rax0三个wifi接口的，

但是此项目lede只有ra0 rai0两个接口。

如何添加额外的wifi接口呢？

**Comment by aiamadeus** [comment]
那你为什么不改 5G 信道呢，rax0 是给 mt7615d 用的。

**Reply by lushangkan** [reply]
5.2Ghz的穿墙信号好，但速度没有5.8Ghz快。
我需要一个速度优先的和一个距离优先的，2.4ghz速度完全不够

**Comment by padavanonly** [comment]
一个无线芯片只能同时发射一个频段的信号。

**Reply by lushangkan** [reply]
Redmi AC2100的lede默认配置是使用mt7615同时发送2.4ghz和5ghz信号的

**Reply by aiamadeus** [reply]
> Redmi AC2100的lede默认配置是使用mt7615同时发送2.4ghz和5ghz信号的

？那只是 mtwifi 接口的名字，实际上 2.4G 是垃圾 mt7603。
